svgdigitizer/electrochemistry/cv.py

```diff
@@ -170,155 +170,21 @@ class CV:
-            ...   </g>
-            ...   <text x="-200" y="330">scan rate: 50 V/s</text>
-            ... </svg>'''))
-            >>> cv = CV(SVGPlot(svg))
-            >>> cv.voltage_dimension
-            Traceback (most recent call last):
-            ...
-            svgdigitizer.exceptions.SVGAnnotationError: The voltage must be on the x-axis in the SVG.
-
-        """
-        dimensions = list(set(["E", "U"]).intersection(self.svgplot.schema.field_names))
-
-        if len(dimensions) == 1:
-            if (
-                self.svgplot.schema.get_field(dimensions[0]).custom["orientation"]
-                == "x"
-            ):
-                return dimensions[0]
-            raise SVGAnnotationError("The voltage must be on the x-axis in the SVG.")
-
-        raise SVGAnnotationError(
-            "No voltage axis or more than one voltage axis found in the SVG."
-        )
-
-    @property
-    def current_dimension(self):
-        r"""
-        The dimension of the current axis given as
-        ``I`` (current) or ``j`` (current density).
-
-        EXAMPLES::
-
-            >>> from svgdigitizer.svg import SVG
-            >>> from svgdigitizer.svgplot import SVGPlot
-            >>> from svgdigitizer.electrochemistry.cv import CV
-            >>> from io import StringIO
-            >>> svg = SVG(StringIO(r'''
-            ... <svg>
-            ...   <g>
-            ...     <path d="M 0 100 L 100 0" />
-            ...     <text x="0" y="0">curve: 0</text>
-            ...   </g>
-            ...   <g>
-            ...     <path d="M 0 200 L 0 100" />
-            ...     <text x="0" y="200">E1: 0 V vs. RHE</text>
-            ...   </g>
-            ...   <g>
-            ...     <path d="M 100 200 L 100 100" />
-            ...     <text x="100" y="200">E2: 1 V vs. RHE</text>
-            ...   </g>
-            ...   <g>
-            ...     <path d="M -100 100 L 0 100" />
-            ...     <text x="-100" y="100">j1: 0 uA / cm2</text>
-            ...   </g>
-            ...   <g>
-            ...     <path d="M -100 0 L 0 0" />
-            ...     <text x="-100" y="0">j2: 1 uA / cm2</text>
-            ...   </g>
-            ...   <text x="-200" y="330">scan rate: 50 V/s</text>
-            ... </svg>'''))
-            >>> cv = CV(SVGPlot(svg))
-            >>> cv.current_dimension
-            'j'
-
-        """
-        dimensions = list(set(["I", "j"]).intersection(self.svgplot.schema.field_names))
-
-        if len(dimensions) == 1:
-            if (
-                self.svgplot.schema.get_field(dimensions[0]).custom["orientation"]
-                == "y"
-            ):
-                return dimensions[0]
-            raise SVGAnnotationError("The current must be on the y-axis in the SVG.")
-
-        raise SVGAnnotationError(
-            "No current axis or more than one current axis found in the SVG."
+    def __init__(self, svgplot, metadata=None, measurement_type="CV", si_units=False):
```

Comment:
```suggestion
    def __init__(self, svgplot, metadata=None, measurement_type="CV", force_si_units=False):
```

Otherwise it sounds like we want non-SI units.

Or at least change the default to ``None``, so it sounds more neutral.